packages/adp-tooling/src/preview/routes-handler.ts

```diff
@@ -112,4 +117,104 @@ export default class RoutesHandler {
             next(e);
         }
     };
+
+    /**
+     * Handler for reading all controller js files from the workspace.
+     *
+     * @param _ Request
+     * @param res Response
+     * @param next Next Function
+     */
+    public handleReadAllControllers = async (_: Request, res: Response, next: NextFunction) => {
```

Comment:
`handleReadAllControllers` and `handleReadAllFragments` are quite similar, does it make sense to have a common function that both call with paramters?